spec/services/submissions_inviter_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe SubmissionsInviter do
+  describe '#invite' do
+    let(:submission) { instance_double Submission }
+    let(:message_delivery) { instance_double ActionMailer::MessageDelivery }
```

Comment:
I wouldn't mock rails code, I would just check `ActionMailer::Base.deliveries.count` after sending the email.